File: CalcLau em Java/Netbeans Project/CalcLau/src/backlogic/CalcOp.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package backlogic;

/**
 *
 * @author Lau
 */
public class CalcOp {
        private static String resultado;
        private static long valor;
        private static long valor1;
        private static long valor2;
        
        
    public static void Soma(){
        valor = valor2 + valor1;
        resultado = String.valueOf(valor);
    }
    
    public static void Sub(){
        valor = valor2 - valor1;
        resultado = String.valueOf(valor);
    }
    
    public static void Multi(){
        valor = valor2 * valor1;
        resultado = String.valueOf(valor);
    }
    
    public static void Div(){
        valor = valor2 / valor1;
        resultado = String.valueOf(valor);
    }

    public static String getResultado() {
        return resultado;
    }

    public static void setValor1(long valor1) {
        CalcOp.valor1 = valor1;
    }

    public static void setValor2(long valor2) {
        CalcOp.valor2 = valor2;
    }
}

File: CalcLau em C#/Visual Studio/CalcLau/CalcLau/CalcOp.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CalcLau
{
    class CalcOp
    {
        private static string resultado;
        private static long valor;
        private static long valor1;
        private static long valor2;


        public static void Soma()
        {
            valor = valor2 + valor1;
            resultado = valor.ToString();
        }

        public static void Sub()
        {
            valor = valor2 - valor1;
            resultado = valor.ToString();
        }

        public static void Multi()
        {
            valor = valor2 * valor1;
            resultado = valor.ToString();
        }

        public static void Div()
        {
            valor = valor2 / valor1;
            resultado = valor.ToString();
        }

        public static string getResultado()
        {
            return resultado;
        }

        public static void setValor1(long valor1)
        {
            CalcOp.valor1 = valor1;
        }

        public static void setValor2(long valor2)
        {
            CalcOp.valor2 = valor2;
        }
    }
}
